MeshLib/MeshSurfaceExtraction.cpp

```diff
@@ -223,51 +221,48 @@ void MeshSurfaceExtraction::get2DSurfaceElements(const std::vector<MeshLib::Elem
     }
 }
 
-void MeshSurfaceExtraction::get2DSurfaceNodes(std::vector<MeshLib::Node*> &sfc_nodes, std::size_t n_all_nodes, const std::vector<MeshLib::Element*> &sfc_elements, std::vector<std::size_t> &node_id_map)
+void MeshSurfaceExtraction::get2DSurfaceNodes(
+    std::vector<MeshLib::Node*>& sfc_nodes, std::size_t n_all_nodes,
+    const std::vector<MeshLib::Element*>& sfc_elements,
+    std::vector<std::size_t>& node_id_map)
 {
-    const std::size_t nNewElements (sfc_elements.size());
     std::vector<const MeshLib::Node*> tmp_nodes(n_all_nodes, nullptr);
-    for (std::size_t i=0; i<nNewElements; ++i)
+    for (auto const* elem : sfc_elements)
     {
-        const MeshLib::Element* elem (sfc_elements[i]);
-        for (unsigned j=0; j<elem->getNumberOfBaseNodes(); ++j)
+        for (unsigned j = 0; j < elem->getNumberOfBaseNodes(); ++j)
         {
-            const MeshLib::Node* node (elem->getNode(j));
+            const MeshLib::Node* node(elem->getNode(j));
             tmp_nodes[node->getID()] = node;
         }
     }
-    const std::size_t nNodes (tmp_nodes.size());
-    for (unsigned i=0; i<nNodes; ++i)
+    const std::size_t nNodes(tmp_nodes.size());
+    for (unsigned i = 0; i < nNodes; ++i)
     {
         if (tmp_nodes[i])
         {
             node_id_map[i] = sfc_nodes.size();
-            sfc_nodes.push_back(new MeshLib::Node(tmp_nodes[i]->getCoords(), tmp_nodes[i]->getID()));
+            sfc_nodes.push_back(new MeshLib::Node(*tmp_nodes[i]));
         }
     }
 }
 
-std::vector<GeoLib::Point*> MeshSurfaceExtraction::getSurfaceNodes(const MeshLib::Mesh &mesh, const MathLib::Vector3 &dir, double angle)
+std::vector<MeshLib::Node*> MeshSurfaceExtraction::getSurfaceNodes(
+    const MeshLib::Mesh& mesh, const MathLib::Vector3& dir, double angle)
```

Comment:
Only reformat in the PR. Since `angle` is not given via reference or a pointer the value is not changed. So no need to set it `const`.